src/main/java/mil/dds/anet/database/mappers/PersonPreferenceMapper.java

```diff
@@ -0,0 +1,29 @@
+package mil.dds.anet.database.mappers;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import mil.dds.anet.beans.PersonPreference;
+import org.jdbi.v3.core.mapper.RowMapper;
+import org.jdbi.v3.core.statement.StatementContext;
+
+public class PersonPreferenceMapper implements RowMapper<PersonPreference> {
+
+  @Override
+  public PersonPreference map(ResultSet rs, StatementContext ctx) throws SQLException {
+    PersonPreference p = fillInFields(new PersonPreference(), rs);
+
+    if (MapperUtils.containsColumnNamed(rs, "totalCount")) {
+      ctx.define("totalCount", rs.getInt("totalCount"));
+    }
+    return p;
+  }
+
+  public static PersonPreference fillInFields(PersonPreference p, ResultSet rs)
```

Comment:
Please inline this method, it is only called from `map()`.
You probably also want to map the `createdAt` and `updatedAt` fields.